index.qmd

```diff
@@ -39,73 +40,58 @@ format:
           margin-left: 0
         embed-resources: true
         page-layout: full
-my-custom-stuff:
-   my-reuseable-variable: "I can use this wherever I want in the markdown, and change it in only once place :)"
 ---
 
-## Contents
 
-Some example slides - [also look at example RevealJS slides in the Quarto docs](https://quarto.org/docs/presentations/revealjs/)
+## Welcome
 
-* Non-executable and executable code-blocks
-* bullet points with highlighting 
-* two-column slides 
-* how to include a slide from a separate MD file
-* preview and link to a webpage
+![](img/sainsbury-wellcome-centre.jpg)
 
-## Just a code block, nothing gets executed...
+## Neuroinformatics Unit
 
-... but there is some fancy highlighting
+![](img/niu-team.png)
 
-```{.python code-line-numbers="1|3|4-9"}
-from pathlib import Path
+## Code of Conduct
 
-home_path = Path.home()
-if home_path.exists():
-  data_path = home_path / "data"
-else:
-  pass
-  # raise some error maybe?
-```
+* Reminder: please follow the [code of conduct](https://docs.google.com/document/d/1pQGML9twa-GnqQNDtNRg1P2_Wwzvl1b61A6fLzpY_2Q/edit?tab=t.6jlq0oph6gve)
+* Summary: be kind, respectful and professional
+* Contacts: Adam, Sofia, Niko or Alessandro
 
-## A code block that's actually executed at render-time
+## Schedule
 
-```{python}
-#| echo: true
-#| code-fold: true
+![](img/schedule-2025.png)
 
-from pathlib import Path
+## Building map
 
-print("Hello world")
-```
+![](img/swc-maps.png)
 
-## You can execute code without showing that you have by using #|echo: false
-```{python}
-#| echo: false
+Ask a member of NIU for a quiet space
 
-from pathlib import Path
+## Ask for help
 
-print("Hello world")
-```
+* ask any member of NIU 
+* zulip or NIU
+* work together
 
-{{< include slides/extra_slide.qmd >}}
+## Join the socials
+* Monday: walk to Primrose Hill
+* Tuesday: Celtic folk music in King's Cross
+* Wednesday: pub quiz at SWC
+* Thursday: AlgoChill in Farringdon ([sign up](https://lu.ma/3z1ureee))
 
-## An example image
+## Thanks to our funders {.smaller}
 
-Include an image:
+![](img/sponsor-logos.png){height=50% width=50% fig-align=center}
 
-![](img/bg_logo_wide.png){width=900 fig-align=center}
+* 2025 SSI fellowship "Kickstart a community of practice for open-source tools in animal behaviour" awarded to Niko Sirmpilatze
+* 2025 SSI fellowship "Bridging Bioimaging and Research Software Engineering" awarded to Alessandro Felder
 
+Made possible by
 
-## Link and a preview a webpage:
+![](img/swc-logo.webp){height=30% width=30% fig-align=center}
 
-::: {style="text-align: center; margin-top: 1em"}
-[interoperable Python-based tools for computational neuroanatomy](https://brainglobe.info/index.html){preview-link="true" style="text-align: center"}
-:::
+ 
 
-## Use a variable several times
+![](img/general-funders.webp){height=30% width=30% fig-align=center}
```

Comment:
These 3 appear tiny in comparison to the SWC one. I'd try putting all 4 logos (SWC, Gatbsy, Wellcome, UCL) in one row/image.